arches/app/media/js/viewmodels/workflow.js

```diff
@@ -301,6 +301,14 @@ define([
             });
         };
 
+        this.reverseWorkflowTransactions = function() {
+            $.ajax({
+                type: "POST",
+                url: arches.urls.transaction_reverse(self.id()),
+                context: self,
```

Comment:
Do we need to pass the entire workflow as context in this call?

Reply:
I don't think so since there's no callback on this POST anyway.  Will remove.